### ISSUE TITLE
BusyIndicator : Accessibility screen reader does not identify if busy indicator is set or not

### ISSUE THREAD

**Issue by pppwr7** [SAP/ui5-webcomponents#5414]
**Describe the bug**
Accessibility screen reader does not identify when busy indicator is set or not.

**Isolated Example**
use react library sample https://sap.github.io/ui5-webcomponents-react/?path=/docs/user-feedback-busyindicator--default-story for testing

*

**Comment by nnaydenow** [comment]
Hello @SAP/ui5-webcomponents-topic-rd ,

Could you please check this issue? If busy indicator is activated and focus is inside it screen reader doesn't announce any information.

You can check it in this example: https://codesandbox.io/s/ui5-webcomponents-forked-we2r53?file=/src/index.js

I checked the behaviour in OpenUI5 and there screen reader doesn't announce any information too. Is this the correct behaviour ?

Regards,
Nayden

**Comment by dimovpetar** [comment]
Hello @PriyaPowar ,

To clarify, when you test, when the "Focusable element" div becomes busy, which is the current active element? Is it button, or the "Focusable element" div itself?

Best regards,
Petar

**Comment by pppwr7** [comment]
Hi [dimovpetar](https://github.com/dimovpetar),
Apologies for the very delayed reply.  Our requirement is that we have a dynamic page with filterbar in the header and a table in dynamic page content section. On click of Go button in filterbar, the table part is shown as under a busy indicator so that user does not perform any action son the table while the call is still in progress. Hence, when user clicks on Go button, we want the busy indicator being set and unset to be read.

**Comment by dimovpetar** [comment]
Reopening this issue for further investigation.

**Comment by GerganaKremenska** [comment]
Hello @pppwr7,

As i understand this is not for ui5 web components, but for sapui5 controls correct?

The announcment of busy indicator in both projects will be announced if the focus is on the busy inducator it self.

On pressign "Go" button the focus will not be moved automaticly to the table, so when the user presses "Go" the table will start loading and if they go through the page they will see that the page is loading.

I am closing the issue, as there is no accessibility incident here reagarding ui5 web components.

Best Regards,
Gergana
